src/converters/cover.ts

```diff
@@ -192,7 +235,7 @@ class CoverHandler implements ServiceHandler {
       let didStop = true;
 
       // As long as the update timer is running, we are expecting updates.
-      if (this.updateTimer !== undefined && this.updateTimer.isActive) {
+      if (this.updateTimer !== undefined && this.updateTimer.isActive && !this.modern) {
```

Comment:
This change should not be needed, given that you do not initialise `updateTimer` if you have the motor state information.

```suggestion
      if (this.updateTimer !== undefined && this.updateTimer.isActive) {
```